site/gdocs/components/Byline.tsx

```diff
@@ -0,0 +1,39 @@
+import { getCanonicalUrl } from "@ourworldindata/components"
+import { OwidGdocType } from "@ourworldindata/types"
+import React from "react"
+import { useLinkedAuthor } from "../utils.js"
+
+export const Byline = ({ names }: { names: string[] }) => {
+    return (
+        <>
+            {"By: "}
+            {names.map((name, idx) => (
+                <React.Fragment key={name}>
+                    <LinkedAuthor name={name} />
+                    {idx === names.length - 1
+                        ? ""
+                        : idx === names.length - 2
+                          ? " and "
```

Comment:
Pretty sure we'd want the [oxford comma](https://www.notion.so/owid/Writing-and-style-guide-d51a3739ff8542ca90297fa8de40437c?pvs=4#27f2da94fa8b4cf4a2065944a89d566d) here

Reply:
my thinking behind it was:
- it seems like the main reason for it is to avoid confusion. Here, given that we don't have a real sentence, there isn't any possible confusion
- we currently don't have it in the byline
- it requires special handling when we only have two authors and don't want the oxford comma.

I won't change it for now, but let me know if you see things differently?